<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWidgetClass</class>
 <widget class="QWidget" name="GroupWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>258</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="buttonAddSynonym">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Synonym group</string>
     </property>
     <property name="text">
      <string>Synonyms</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonAddAntonym">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Antonym group</string>
     </property>
     <property name="text">
      <string>Antonyms</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonAddSimilar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Confusing group</string>
     </property>
     <property name="text">
      <string>Confusing</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonDel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Delete group</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/edit_remove.png</normaloff>:/Entrance/Images/edit_remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Related</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Explanation</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyWords.qrc"/>
 </resources>
 <connections/>
</ui>
